src/applications/Machine.java

```diff
@@ -4,50 +4,14 @@
 
 class Machine {
     // data members
-    private LinkedQueue jobQ; // queue of waiting jobs for this machine
-    private int changeTime; // machine change-over time
-    private int totalWait; // total delay at this machine
-    private int numTasks; // number of tasks processed on this machine
-    private Job activeJob; // job currently active on this machine
```

Comment:
This seems like a bad thing. In general I think we want to try to make as many things private as possible.